formstack-baton-requests/cloud-formation.yaml

```diff
@@ -274,6 +274,168 @@ Resources:
+          RESULTS_PATH: !Sub formstack-results/${Stage}
+          PERFORM_FUNCTION_NAME: !Ref PerformFormstackRerLambda
+      MemorySize: 384
+      Runtime: java8
+      Timeout: 120
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref LambdaSecurityGroup
+        SubnetIds: !Ref VpcSubnets
+      Role:
+        !GetAtt FormstackBatonRerLambdaRole.Arn
+
+  PerformFormstackRerLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: EC2Policy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - ec2:CreateNetworkInterface
+                  - ec2:DescribeNetworkInterfaces
+                  - ec2:DeleteNetworkInterface
+                Resource: "*"
+        - PolicyName: LogPolicy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - logs:CreateLogGroup
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource: !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/formstack-baton-perform-rer-lambda-${Stage}:log-stream:*"
+        - PolicyName: S3ListPolicy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:ListBucket
+                Resource: !Sub arn:aws:s3:::${ResultsBucket}
+        - PolicyName: S3WritePolicy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:PutObject
+                  - s3:PutObjectAcl
+                  - s3:GetObject
+                Resource: !Sub arn:aws:s3:::${ResultsBucket}/formstack-results/${Stage}/*
+        - PolicyName: GetParamsPolicy
+          PolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action: ssm:GetParameter
+                Resource:
+                  - !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${FormstackAccountOneTokenPath}
+                  - !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${FormstackAccountTwoTokenPath}
+                  - !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${EncryptionPasswordPath}
+                  - !Sub arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter${BcryptSaltPath}
+
+  PerformFormstackRerLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: Performs RER to Formstack, writing status updates to S3
+      FunctionName:
+        !Sub formstack-baton-perform-rer-lambda-${Stage}
+      Code:
+        S3Bucket: identity-lambda
+        S3Key: !Sub identity/${Stage}/formstack-baton-requests/main.jar
```

Comment:
main.jar? usually we have projectname.jar. not a big deal though.

Reply:
I just did this to keep it in line with the other lambdas that identity has in this project